wiki/Tournaments/112/Summer_2020/en.md

```diff
@@ -0,0 +1,545 @@
+---
+tags:
+- 112servis
+- 112ST
+- 112ST2020
+---
+
+# 112's Summer Tournament 2020
+
+![112ST2020 banner](img/banner.png "112's Summer Tournament 2020")
+
+The **112's Summer Tournament 2020** (***112ST2020***) was a double-elimination 1v1 osu! tournament hosted by ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470) and ![][flag_TR] [Sibyl](https://osu.ppy.sh/users/10440852). The tournament was only allowed for Turkish-speaking users regardless of their country flags. It was the 6th installment of 112's Tournaments series.
```

Comment:
```suggestion
The **112's Summer Tournament 2020** (***112ST2020***) was a double-elimination 1v1 osu! tournament hosted by ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470) and ![][flag_TR] [Sibyl](https://osu.ppy.sh/users/10440852). The tournament was only allowed for Turkish-speaking users regardless of their country flags. It was the sixth installment of 112's Tournaments series.
```

Reply:
Well, @TPGPL requested to change that one before in [one of his suggestions](https://github.com/ppy/osu-wiki/pull/4256#discussion_r495589653). Why do you want to revert it now? I'm confused.

Reply:
looks like the formatting for this sentence is pretty inconsistent, as tournaments have it both way (e.g. OWC uses the number format for its installments while TWC uses text format)